.gitmodules

```diff
@@ -1,8 +1,8 @@
 [submodule "vendor/nim-eth"]
 	path = vendor/nim-eth
-	url = https://github.com/status-im/nim-eth.git
+	url = https://github.com/kaiserd/nim-eth.git
```

Comment:
Can we get this to point to the branch `add-selectable-protocol-id-static` on status-im nim-eth instead? Easier to keep up to date with permissions etc as opposed to at your individual account. You should have permissions to push branches

Is there an issue for tracking this upstream?